pkg/config/cos.go

```diff
@@ -606,7 +594,7 @@ func updateBridge(stage *yipSchema.Stage, name string, mgmtNetwork *Network) err
 		Group:       0,
 	})
 
-	preUpScript, err := render("wicked-setup-bridge.sh", MgmtBondInterfaceName)
+	preUpScript, err := render("wicked-setup-bridge.sh", mgmtNetwork.VlanID)
```

Comment:
If user does not input mgmt vlan id, namely untagged network, is mgmtNetwork.VlanID == 0 ? what will hapen? thanks.

Reply:
The `BRIDGE_PRIORITY=0` will show in `ifcfg-mgmt-br` file which will not be effective. Since it's 0, which is not a valid vlan id, the `bridge add vid` command will not be in the setup_bridge.sh file. So the `pre-up` script will just set back to the default vlan setting, 1 for PVID, and 2 - 4094 for vid.